code/game/objects/items/weapons/manuals.dm

```diff
@@ -838,7 +838,7 @@
 
 /obj/item/book/manual/chef_recipes
     name = "Chef Recipes"
-    icon_state = "cooked_book"
+    icon_state = "CookBook"
```

Comment:
Not 100% sure what the convention on icon names is, but 99% of the icons have a `snake_case` name, so I'd prefer if that consistency was kept.

Reply:
updated to cook_book